src/pages/index.tsx

```diff
@@ -20,12 +22,13 @@ export default function Home() {
             <div className={styles.formFields}>
               <h1 className={styles.title}>sbtmaker</h1>
               {
-                !tokenURI ? <LookAndFeel setMenu={setMenu} setTokenURI={setTokenURI} />
-                : <Deploy setMenu={setMenu} tokenURI={tokenURI} />
+                finalStep ? <Success transactionHash={transactionHash} /> :
+                  !tokenURI ? <LookAndFeel setMenu={setMenu} setTokenURI={setTokenURI} />
+                  : <Deploy setMenu={setMenu} tokenURI={tokenURI} transactionHash={transactionHash} setTransactionHash={setTransactionHash} setFinalStep={setFinalStep} />
               }
```

Comment:
C'est très difficile à lire là. 

Peux tu installer [prettier](https://www.npmjs.com/package/prettier) et le run sur tous les fichiers en faisant `prettier write`. 

Voici un tuto https://levelup.gitconnected.com/how-to-format-all-files-in-a-directory-with-prettier-5f0ff5f4ffb2